Products/CMFPlone/patterns/__init__.py

```diff
@@ -242,6 +241,8 @@ def tinymce(self):
 
         image_types = settings.image_objects or []
         folder_types = settings.contains_objects or []
+
+        site_path = generator.portal_url.replace(self.request['SERVER_URL'], '')
```

Comment:
The point isn't only to strip off the domain. The point is to get the absolute path to the site. This is different depending on if it is using virtualhostmonster or not.